ipynb/utils/executor_example.ipynb

```diff
@@ -68,10 +69,41 @@
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "05:06:03  INFO    :         Target - Using base path: /home/bjackman/sources/lisa\n",
+      "05:06:03  INFO    :         Target - Loading custom (inline) target configuration\n",
+      "05:06:03  INFO    :         Target - Devlib modules to load: ['bl', 'cpufreq']\n",
+      "05:06:03  INFO    :         Target - Connecting linux target:\n",
+      "05:06:03  INFO    :         Target -   username : brendan\n",
+      "05:06:03  INFO    :         Target -       host : 192.168.0.1\n",
+      "05:06:03  INFO    :         Target -   password : \n",
+      "05:06:03  INFO    :         Target - Connection settings:\n",
+      "05:06:03  INFO    :         Target -    {'username': 'brendan', 'host': '192.168.0.1', 'password': ''}\n",
+      "05:06:09  INFO    :         Target - Initializing target workdir:\n",
+      "05:06:09  INFO    :         Target -    /home/brendan/devlib-target\n",
+      "05:06:12  INFO    :         Target - Topology:\n",
+      "05:06:12  INFO    :         Target -    [[0, 3, 4, 5], [1, 2]]\n",
+      "05:06:17  INFO    :         FTrace - Enabled tracepoints:\n",
+      "05:06:17  INFO    :         FTrace -   sched_switch\n",
+      "05:06:17  INFO    :         FTrace -   sched_wakeup\n",
+      "05:06:17  INFO    :         FTrace -   sched_wakeup_new\n",
+      "05:06:17  INFO    :         FTrace -   cpu_frequency\n",
+      "05:06:17  WARNING :        TestEnv - Wipe previous contents of the results folder:\n",
+      "05:06:17  WARNING :        TestEnv -    /home/bjackman/sources/lisa/results/ExecutorExample\n",
+      "05:06:17  INFO    :        TestEnv - Set results folder to:\n",
+      "05:06:17  INFO    :        TestEnv -    /home/bjackman/sources/lisa/results/ExecutorExample\n",
+      "05:06:17  INFO    :        TestEnv - Experiment results available also in:\n",
+      "05:06:17  INFO    :        TestEnv -    /home/bjackman/sources/lisa/results_latest\n"
+     ]
+    }
+   ],
    "source": [
     "# Setup a target configuration\n",
-    "my_target_conf = {\n",
+    "env = TestEnv({\n",
```

Comment:
Interesting... we should probably use this inline version every time we do not need to access the configuration from somewhere else code in the notebook...